rfcs/imported/0000-dcp-media-types.md

```diff
@@ -0,0 +1,149 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# DCP Media Types
+
+DISCLAIMER: Please note that this RFC has been imported its original Google Doc and went through a design review process prior to the implementation of the current [RFC Process](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md). The original document is [here](https://docs.google.com/document/d/1TqihrgXjct9aDmTJO52_gE2WlpFysB1OkG9C8exmWTw/edit#heading=h.87ix45a71erf). Please be aware that the contents below may potentially be out-of-date as the last-modified date of the original Google Document is October 20th, 2017. 
```

Comment:
Your "out of date" comment seems unnecessary.  They same could be said of all RFCs the day after they are written.
And AFAICT this information is all still current at this time.

Reply:
Agreed; we should be checking for accuracy on import.  I think the important thing to note  is not that is was in google docs, but it was approved by an older process.

It would be good to update the google doc to point to the RFC.

Reply:
@sampierson Deleted the last sentence. Thanks.

@diekhans I have an earlier sentence in that paragraph "and was approved by a design review process prior to the implementation of the current RFC Process."

@diekhans @sampierson I will let the original author decide whether to point the Google Doc to the RFC :)